File: ThisIsCodingTest/이것이코딩테스트다_11장_모험가길드_3.py
N = int(input())
adventurers = list(map(int, input().split()))
adventurers.sort()

total_groups = 0
members = 0
for adventurer in adventurers:
    members += 1
    if adventurer == members:
        total_groups += 1
        members = 0

print(total_groups)

File: this-is-coding-test/이것이코딩테스트다_11장_모험가길드_3.js
input = '5\n2 3 1 2 2';

input = input.split('\n');
N = parseInt(input[0]);
adventurers = input[1].split(' ').map(v => parseInt(v))
adventurers.sort((a, b) => a - b);

max_group_cnt = 0;
cnt = 0;

for (const adventurer of adventurers) {
  cnt += 1;

  if (adventurer !== cnt) {
    continue;
  }

  max_group_cnt += 1;
  cnt = 0;
}

console.log(max_group_cnt);
